doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -136,6 +136,8 @@ nRF9160: Asset Tracker v2
 
 * Updated:
 
+  * Updated default value of Kconfig option :kconfig:option:`CONFIG_DATA_ACTIVE_TIMEOUT_SECONDS` to 300 seconds.
+
   * Enabled link time optimization to reduce the flash size of the application.
```

Comment:
Removed updated since we have "Updated" as section heading.
```suggestion
  * Default value of the Kconfig option :kconfig:option:`CONFIG_DATA_ACTIVE_TIMEOUT_SECONDS` is changed to 300 seconds.
  * Enabled link time optimization to reduce the flash size of the application.
```

Reply:
Done